akaibot.py

```diff
@@ -57,22 +58,33 @@ async def on_message(self, message: discord.Message):
         await self.command.handle(message)
 
     async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
-        self.logger.debug('Recived reaction add info')
-        self.logger.debug(f'User {payload.member.display_name} reacted with {payload.emoji.name} '
-                         f'to message {payload.message_id}')
+        member = self.get_guild(payload.guild_id).get_member(payload.user_id)
+        self.logger.debug('Received reaction add info')
+
+        if payload.emoji.name[:-1] == "thanks":
```

Comment:
Na Twoim serwie testowym to może działać, bo dodałeś emotki pasujące do tego, na AKAI-owym serwie w takim stanie to nie zadziała. Spróbuj gdzieś to wyrzucić, żeby ten if nie był tak hardcoded